Marlin/Configuration.h

```diff
@@ -255,6 +275,12 @@ Here are some standard links for getting your machine calibrated:
 //    #define  DEFAULT_bedKi 1.41
 //    #define  DEFAULT_bedKd 1675.16
 
+//24V PCB heater 4 ohm
+//from PID autotune
+    #define  DEFAULT_bedKp 107.41
```

Comment:
If I understand correctly this overrides the definitions uncommented above. We should probably comment those ones out or replace them with these.